string conversion warning

```
src/web/config_load.cc:227:58: warning: implicit conversion turns string literal into bool: 'const char [9]' to 'bool' [-Wstring-conversion]
        setValue(item, entry->getType() == TR_External ? "external" : "none");
        ~~~~~~~~                                         ^~~~~~~~~~
src/web/config_load.cc:227:71: warning: implicit conversion turns string literal into bool: 'const char [5]' to 'bool' [-Wstring-conversion]
        setValue(item, entry->getType() == TR_External ? "external" : "none");

```

ping @KarlStraussberger

--- a/src/web/config_load.cc
+++ b/src/web/config_load.cc
@@ -224,7 +224,7 @@ void web::configLoad::process()
 
         item = values.append_child("item");
         createItem(item, cs->getItemPath(pr, ATTR_TRANSCODING_PROFILES, ATTR_TRANSCODING_PROFILES_PROFLE, ATTR_TRANSCODING_PROFILES_PROFLE_TYPE), cs->option, ATTR_TRANSCODING_PROFILES_PROFLE_TYPE);
-        setValue(item, entry->getType() == TR_External ? "external" : "none");
+        setValue(item, std::string((entry->getType() == TR_External) ? "external" : "none"));
 
         item = values.append_child("item");
         createItem(item, cs->getItemPath(pr, ATTR_TRANSCODING_PROFILES, ATTR_TRANSCODING_PROFILES_PROFLE, ATTR_TRANSCODING_PROFILES_PROFLE_MIMETYPE), cs->option, ATTR_TRANSCODING_PROFILES_PROFLE_MIMETYPE);

